File: app/src/main/java/ladsoft/com/popularmoviesapp/model/MovieVideo.java
package ladsoft.com.popularmoviesapp.model;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class MovieVideo {

    @Expose @SerializedName("id") private String id;
    @Expose @SerializedName("iso_6391_1") private String iso6391;
    @Expose @SerializedName("iso_3166_1") private String iso31661;
    @Expose @SerializedName("key") private String key;
    @Expose @SerializedName("name") private String name;
    @Expose @SerializedName("site") private String site;
    @Expose @SerializedName("size") private long size;
    @Expose @SerializedName("type") private String type;

    public String getId() {
        return id;
    }

    public String getIso6391() {
        return iso6391;
    }

    public String getIso31661() {
        return iso31661;
    }

    public String getKey() {
        return key;
    }

    public String getName() {
        return name;
    }

    public String getSite() {
        return site;
    }

    public long getSize() {
        return size;
    }

    public String getType() {
        return type;
    }
}

File: app/src/main/java/ladsoft/com/popularmoviesapp/model/MovieVideo.kt
package ladsoft.com.popularmoviesapp.model


import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class MovieVideo (@Expose @SerializedName("id") val id: String = "",
    @Expose @SerializedName("iso_6391_1") val iso6391: String = "",
    @Expose @SerializedName("iso_3166_1") val iso31661: String = "",
    @Expose @SerializedName("key") val key: String = "",
    @Expose @SerializedName("name") val name: String = "",
    @Expose @SerializedName("site") val site: String = "",
    @Expose @SerializedName("size") val size: Long = 0.toLong(),
    @Expose @SerializedName("type") val type: String = "")
